modules/advices.go

```diff
@@ -0,0 +1,40 @@
+package modules
+
+import (
+	"fmt"
+	"net/http"
+	"io/ioutil"
+	"github.com/tidwall/gjson"
```

Comment:
I think the usage of this library is useless, you can just unmarshal the content.

Reply:
I used this lib to avoid having a nested struct. If you are ok with that I have no problem removing it.

Reply:
You can do that without a structure. Just create a `map[string]interface{}` variable and unmarshal the content inside it.
Then you can access elements by doing `myVar["field"]`